src/main/webapp/WEB-INF/tags/admin/search/studentRow.tag

```diff
@@ -111,8 +120,11 @@
       <c:if test="${not empty student.publishedFeedbackSessions}">
         <c:forEach items="${student.publishedFeedbackSessions}" var="session">
           <li class="list-group-item list-group-item-success">
-            <strong>${session.fsName}</strong>
-            <input value="${session.link}" readonly class="form-control">
+            <search:emailFormFields linkTitle="${session.fsName}"
+                                    relatedLink="${session.link}"
+                                      subjectType="Feedback session results published"
+                                      sessionStatus="Published"
+                                      student="${student}"/>
```

Comment:
These attributes appear to have become misaligned after one of the commits. However (as stated in the main review comment) you might end up replacing the contents here anyway.